lib/active_shipping/carriers/canada_post_pws.rb

```diff
@@ -294,6 +294,16 @@ def parse_rates_response(response, origin, destination)
       CPPWSRateResponse.new(true, "", {}, :rates => rates)
     end
 
+    def price_from_node(node, exclude_tax)
+      price = node.at('price-details/due').text
+      return price unless exclude_tax
+      taxes = 0
+      node.at('price-details/taxes').children.each do |node|
+        taxes += Package.cents_from(node.text) if node.elem?
```

Comment:
Why not

```ruby
children = node.at('price-details/taxes').children
tax_total_cents = children.sum { |node| node.elem? ? Package.cents_from(node.text) : 0 }
```

I also renamed the variable `taxes` to `taxes_total_cents` because let's be clear in what the value it represents is.